spring-cloud-tencent-examples/polaris-config-example/src/main/resources/bootstrap.yml

```diff
@@ -5,7 +5,7 @@ spring:
     name: polaris-config-example
   cloud:
     polaris:
-      address: grpc://183.47.111.80:8091
+      address: grpc://127.0.0.1:8091
```

Comment:
183.47.111.80 is polaris demo environment. (http://14.116.241.63:8080/#/). So user can run example with one click.

So dont change to 127.0.0.1 ?